wagon-provider-api/src/main/java/org/apache/maven/wagon/AbstractWagon.java

```diff
@@ -36,21 +49,6 @@
 import org.apache.maven.wagon.resource.Resource;
 import org.codehaus.plexus.util.IOUtil;
 
-import java.io.File;
```

Comment:
Skip the reformat, unrelated.

Reply:
artifact of netbean's fix imports. i'll roll it back